org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/Gateway.java

```diff
@@ -8,9 +8,35 @@
  *
  * Contributors:
  *     Sebastian Wagner - initial API and implementation
+ *     ZTE - support of more gateways
+ *******************************************************************************/
+/*******************************************************************************
```

Comment:
I think, this line and the next one is obsolete as the modifications are listed in line 11.

Please also add your company after line 2:

```
 * Copyright (c) 2015-2017 University of Stuttgart.
 * Copyright (c) 2017 ZE Corporation
```

This is the style the Eclipse Software Foundation demands (if I understand it correctly.)

Could you do that for all files?